packages/components/inputs/time-input/src/time-input.tsx

```diff
@@ -212,7 +235,7 @@ TimeInput.toLocaleTime = (time, locale) => {
     minute: 'numeric',
     // only show seconds when time contains seconds
     second: parsedTime.seconds > 0 ? 'numeric' : undefined,
-  };
+  } as const;
```

Comment:
What's this for?

Reply:
I was looking at narrowing down the literal types from the `options` expression using const assertion. I think I can use the types from `Intl.DateTimeFormatOptions` as the `options ` type definition instead.